sigmastate/src/main/scala/org/ergoplatform/ErgoLikeContext.scala

```diff
@@ -228,46 +229,84 @@ object ErgoLikeContext {
 /** When interpreted evaluates to a ByteArrayConstant built from Context.minerPubkey */
 case object MinerPubkey extends NotReadyValueByteArray with ValueCompanion {
   override def opCode: OpCode = OpCodes.MinerPubkeyCode
+  /** Cost of calling Context.minerPubkey Scala method. */
+  override val costKind = FixedCost(20)
   override val opType = SFunc(SContext, SCollection.SByteArray)
   override def companion = this
+  protected final override def eval(env: DataEnv)(implicit E: ErgoTreeEvaluator): Any = {
+    addCost(this.costKind)
+    E.context.minerPubKey
+  }
 }
 
 /** When interpreted evaluates to a IntConstant built from Context.currentHeight */
 case object Height extends NotReadyValueInt with ValueCompanion {
   override def companion = this
   override def opCode: OpCode = OpCodes.HeightCode
+  /** Cost of: 1) Calling Context.HEIGHT Scala method. */
+  override val costKind = FixedCost(26)
```

Comment:
I found the table very inconvenient for many reasons: 
1) It is useful to have all operation metadata to be in a single operation descriptor (ValueCompanion plays this role). So, the table can be generated (see SpecGen which generates tables for ErgoTree Spec document, which is done in #643, but not yet merged). 
2) the cost descriptors are out of context there and also "why the costs are separate and other operation metadata is not?" 
3) in many cases the code is implemented in generic way using ValueCompanion type. (See all its usages). So in any case costKind property will be necessary, so having costKind property in each operation descriptor (ValueCompanion) and keep cost values somewhere else is not convenient.
4) Cost parameters are also reported by Profiler after SigmaDslSpecification is run. The report contains the table of all cost parameters in comparison to the actual execution time. The table is collected during tests execution and output to the console in afterAll() method. This will be in the next incremental PR along with all the tests for new Evaluator